### ISSUE TITLE
Missing role on menu item

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15336]
Missing role on menu item

-   **Severity:** Low
-   **Affected Populations:**
    -   Blind
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   Windows - Screen Reader
    -   Mac - VoiceOver
    -   Android - TalkBack
    -   iOS - VoiceOver
-   **Components affected:**
    -   Tools and Options

#### Issue description

The "Copy All Content" menu item is not announced to screen readers as
a menu item, it's announced as a button, because it lacks the
"menuitem" role that the other items have. This also affects the way
the menu as a whole is described, since the announced number of items
does not include this button.

Assistive technologies rely on role information to understand the
purpose of content, and if this information is incomplete or inaccurate
it may create a confusing experience for users.

##### Issue Code

```
    <button class="components-button components-clipboard-button components-menu-item__button" type="button">Copy All Content</button>
```

#### Remediation Guidance

Add `role="menuitem"` to the `<button>` element, consistent with
the other items in the menu.

##### Recommended Code

```
    <button class="components-button components-clipboard-button components-menu-item__button" role="menuitem" type="button">Copy All Content</button>

```

#### Relevant standards

-   [4.1.2 Name, Role, Value
    (Level A)](https://www.w3.org/TR/WCAG20/#ensure-compat-rsv)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-67 in Tenon's report
